### PR TITLE
PubKeySignRegisterTest failures in WebAuthn tests

### LINKED ISSUE #9693: PubKeySignRegisterTest failures in WebAuthn tests
### Describe the bug

The same for the passwordless test case in the test.

See: https://keycloak-jenkins.com/job/universal-test-pipeline-server/1177/testReport/org.keycloak.testsuite.webauthn.registration/PubKeySignRegisterTest/

```
org.openqa.selenium.UnhandledAlertException: 
unexpected alert open: {Alert text : Please input your registered authenticator's label}
  (Session info: chrome=90.0.4430.72): Please input your registered authenticator's label
Build info: version: '4.1.0', revision: '87802e897b'
System info: host: 'paas.psi.redhat.com', ip: '127.0.0.1', os.name: 'Linux', os.arch: 'amd64', os.version: '4.18.0-240.22.1.el8_3.x86_64', java.version: '1.8.0_311'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [37397bc8984a6b2be0116cac184c9124, findElement {using=id, value=kc-try-again}]
Capabilities {acceptInsecureCerts: true, browserName: chrome, browserVersion: 90.0.4430.72, chrome: {chromedriverVersion: 90.0.4430.24 (4c6d850f087da..., userDataDir: /tmp/.com.google.Chrome.yHXDPT}, goog:chromeOptions: {debuggerAddress: localhost:40863}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), se:cdp: ws://localhost:40863/devtoo..., se:cdpVersion: 90.0.4430.72, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 37397bc8984a6b2be0116cac184c9124
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:131)
	at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:164)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:139)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:559)
	at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
	at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:383)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:375)
	at org.jboss.arquillian.graphene.enricher.WebElementUtils.dropProxyAndFindElement(WebElementUtils.java:160)
	at org.jboss.arquillian.graphene.enricher.WebElementUtils.findElement(WebElementUtils.java:62)
	at org.jboss.arquillian.graphene.enricher.SearchContextInterceptor.intercept(SearchContextInterceptor.java:46)
	at org.jboss.arquillian.graphene.proxy.InvocationContextImpl.invoke(InvocationContextImpl.java:87)
	at org.jboss.arquillian.graphene.enricher.StaleElementInterceptor$1.apply(StaleElementInterceptor.java:47)
	at org.jboss.arquillian.graphene.enricher.StaleElementInterceptor$1.apply(StaleElementInterceptor.java:43)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	at org.jboss.arquillian.graphene.wait.WebDriverWaitImpl.until(WebDriverWaitImpl.java:97)
	at org.jboss.arquillian.graphene.enricher.StaleElementInterceptor.intercept(StaleElementInterceptor.java:43)
	at org.jboss.arquillian.graphene.proxy.InvocationContextImpl.invoke(InvocationContextImpl.java:87)
	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$2.call(GrapheneContextualHandler.java:241)
	at org.jboss.arquillian.graphene.context.BrowserActions.performAction(BrowserActions.java:62)
	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.invoke(GrapheneContextualHandler.java:237)
	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.intercept(GrapheneContextualHandler.java:263)
	at org.openqa.selenium.chrome.ChromeDriver$$EnhancerByGraphene$$f0e7a83f.findElement(<generated>)
	at org.keycloak.testsuite.webauthn.pages.WebAuthnErrorPage.isCurrent(WebAuthnErrorPage.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jboss.arquillian.graphene.proxy.GrapheneProxyHandler.invokeReal(GrapheneProxyHandler.java:129)
	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$1.invoke(GrapheneContextualHandler.java:169)
	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler$2.call(GrapheneContextualHandler.java:241)
	at org.jboss.arquillian.graphene.context.BrowserActions.performAction(BrowserActions.java:62)
	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.invoke(GrapheneContextualHandler.java:237)
	at org.jboss.arquillian.graphene.proxy.GrapheneContextualHandler.intercept(GrapheneContextualHandler.java:263)
	at org.keycloak.testsuite.webauthn.pages.WebAuthnErrorPage$$EnhancerByGraphene$$fb4df33f.isCurrent(<generated>)
	at org.keycloak.testsuite.webauthn.registration.PubKeySignRegisterTest.assertPublicKeyAlgorithms(PubKeySignRegisterTest.java:90)
	at org.keycloak.testsuite.webauthn.registration.PubKeySignRegisterTest.publicKeySignaturesRSA(PubKeySignRegisterTest.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
```

### Version

17.0.0

### PR DESCRIPTION
**Pull request by mabartos**
Fixes #9693

Detailed analysis was made by @rmartinc 

The test case works with Chrome versions <120. For the latest releases, a bug[1][2] is present there and affecting the tool for the virtual authenticators - used for WebAuthn testing in Keycloak. 

For the affected test case, we don't really care about the specific RS; it just needs to be invalid in terms of empty WebAuthn configuration. 

For proper behavior of the test case in older and newer chromedrivers, the RS384 is used. 

In summary, it's only test-related and not a bug in Keycloak. 

[1] https://bugs.chromium.org/p/chromium/issues/detail?id=1489492&q=RS256&can=1
[2] https://chromiumdash.appspot.com/commit/0fab259d3bfb2c50b7546aa07fdfd720f0810289

### CODE REVIEW

**Comment by mabartos** [comment]
@rmartinc @miquelsi Can you check it please? Thanks

**Review by miquelsi — APPROVED** [review]

**Review by rmartinc — APPROVED** [review]
Thanks @mabartos! LGTM as we have talked!

**Review by ahus1 — APPROVED** [review]
Approving as of @rmartinc's review.
